elections/uk/management/commands/uk_create_elections_from_every_election.py

```diff
@@ -0,0 +1,193 @@
+from __future__ import print_function, unicode_literals
+
+import requests
+
+from django.core.management.base import BaseCommand
+from django.conf import settings
+
+
+from candidates.models import (
+    AreaExtra, OrganizationExtra, PartySet, PostExtra, PostExtraElection
+)
+from elections.models import AreaType, Election
+from popolo.models import Area, Organization, Post
+
+
+class Command(BaseCommand):
+    help = 'Create posts and elections from a CSV file at a URL'
+    EE_BASE_URL = getattr(
+        settings, "EE_BASE_URL", "https://elections.democracyclub.org.uk/")
+
+    def handle(self, *args, **options):
+        # TODO Consider filtering on future or current elections?
+        url = "{}api/elections?group_type=organisation".format(
+            self.EE_BASE_URL)
+        next_page = True
+        while next_page:
+            req = requests.get(url)
+            data = req.json()
+            self.process_results(data['results'])
+            next_page_url = data.get('next')
+            if next_page_url:
+                url = next_page_url
+            else:
+                next_page = False
+
+    def process_results(self, results):
+        for election in results:
+            # TODO Support elections with TMP IDs somehow
+            if election.get('election_id'):
+                if election.get('group_type') == "organisation":
+                    self.process_election(election)
+
+    def process_election(self, election_dict):
+        """
+        Create the elections
+        """
+        print(election_dict['election_id'])
+        election_model = self.get_election(election_dict)
+
+        organization_extra = self.get_or_create_organisation(
+            election_model, election_dict)
+
+        if election_dict['children']:
+            for division_id in election_dict['children']:
+                self.add_single_division(
+                    election_model, organization_extra, division_id)
+        else:
+            # This is an election to the org directly, with no divisions
+            # e.g. a mayor or PCC
+            self.add_area(
+                election_dict, election_model, organization_extra)
+
+    def get_election(self, election_dict):
+        election_id = election_dict['election_id']
+        election_date = election_dict['poll_open_date']
+
+        # TODO Set current from the API
+        current = True
+
+        return Election.objects.update_or_create(
+            slug=election_id,
+            election_date=election_date,
+            current=current,
+            defaults={
+                "name": election_dict['election_title'],
+                "for_post_role": election_dict['elected_role'],
+                "candidate_membership_role": "Candidate",
+                "show_official_documents": True,
```

Comment:
This is a good point, and needs a new feature in EE https://trello.com/c/JGrBsXj9/895-support-voting-systems-in-ee